src/rcms/fm/app/level1/HCALlevelOneEventHandler.java

```diff
@@ -187,7 +191,7 @@ else if (useResetForRecover.equals("true")) {
             RunKeySetting.put(new StringT("snippet")   ,new StringT(nodes.item(i).getAttributes().getNamedItem("snippet"   ).getNodeValue()));
           }
           else{
-            String errMessage="Cannot find attribute snippet in this Runkey"+runkeyName+", check the RunConfig entry in userXML!";
+            String errMessage="Cannot find attribute snippet in this Runkey"+runkeyName+", check the LocalRunkey entry in userXML!";
```

Comment:
I think RunConfig here is correct.

Reply:
RunConfig is correct inasmuch as it correctly matches what appears in the grandmaster, but I would argue that we should change the name both in the code (here) and in the grandmasters to something other than RunConfig. Here I chose the name "LocalRunKey" -- if we agree that this is a decent name then we should keep this change and also change what appears in the grandmaster, as per #35 and as per the comment added to the code here: https://github.com/HCALRunControl/levelOneHCALFM/compare/xmlCleanupAndValidation#diff-f7747803ec2cfd6e1d9406833ed3f9d4R179